### ISSUE TITLE
Accessibility update link 

### ISSUE THREAD

**Issue by Arhell** [neovim/neovim.github.io#265]
![Screenshot_1](https://user-images.githubusercontent.com/26163841/159114036-c490e61c-5023-4ec6-a128-bac689861f3c.png)

**Comment by jamessan** [comment]
@Arhell You'll need to provide more information than just a screenshot.  I have no idea what's wrong based on a picture.

**Comment by Arhell** [comment]
> @Arhell You'll need to provide more information than just a screenshot. I have no idea what's wrong based on a picture.

**link-name**
Links must have accessible names.

**Why it matters**
An accessible name is a word or phrase coded in a way that assistive technologies can associate it with a specific user interface object. Assistive technologies can then refer to the object by name, not just by type. When a link doesn’t have an accessible name, people who use assistive technologies have no way of knowing its purpose.

From plugin Accessibility Insights for Web
